lib/matplotlib/dviread.py

```diff
@@ -510,7 +511,7 @@ class DviFont(object):
        Name of the font as used internally by TeX and friends. This
        is usually very different from any external font names, and
        :class:`dviread.PsfontsMap` can be used to find the external
-       name of the font.
+       name of the font. ASCII bytestring.
```

Comment:
If we can't use numpydoc/napoleon-style docstrings I would at least put the type at a more prominent place.